### PR TITLE
Add weprem ulx command

### PR DESCRIPTION
**Pull request by wrefgtzweve**
Kanban card: https://kanban.cfcservers.org/b/zTdv2hrDmprDAKfxz/cfc-public/gRjyNtZpejyDHCzgj

### CODE REVIEW

**Comment by wrefgtzweve** [comment]
image

File: lua/ulx/modules/sh/cfc_weprem.lua [hunk 1]
```diff
@@ -0,0 +1,24 @@
+CFCUlxCommands.weprem = CFCUlxCommands.weprem or {}
+local cmd = CFCUlxCommands.weprem
+local CATEGORY_NAME = "Utility"
+
```
Comment (brandonsturgeon) [inline lua/ulx/modules/sh/cfc_weprem.lua:4]:
```suggestion
local IsValid = IsValid

```

File: lua/ulx/modules/sh/cfc_weprem.lua [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+function cmd.weprem( callingPlayer, targetPlayers )
+    local count = 0
+
+    for _, entity in pairs( ents.GetAll() ) do
```
Comment (brandonsturgeon) [inline lua/ulx/modules/sh/cfc_weprem.lua:8]:
```suggestion
    for _, entity in ipairs( ents.GetAll() ) do
```

**Review by brandonsturgeon — CHANGES_REQUESTED** [review]

File: lua/ulx/modules/sh/cfc_weprem.lua (hunk 1 above)
Comment (wrefgtzweve) [reply lua/ulx/modules/sh/cfc_weprem.lua:4]: Why localize isvalid here?

**Comment by brandonsturgeon** [comment]
Please give this one last test and then we'll merge it 👍

**Comment by wrefgtzweve** [comment]
image
Yup, works fine!

**Review by brandonsturgeon — APPROVED** [review]

**Pull request merged**